atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/restore/V1TransactionsTableRangeDeleter.java

```diff
@@ -47,6 +48,7 @@ public V1TransactionsTableRangeDeleter(
     }
 
     @Override
+    @AllowedRangeRequest(justification = "???")
```

Comment:
any context on what's calling this/why doing this deletion is ok?

Reply:
Yes, only used on the restore path and for the transaction table, which in normal operation does not have any tombstones written. (Yes, there is a risk if you restore a stack twice in succession with no intervening compaction; should we make an internal note of this on SOPs or similar?)